### DISCUSSION TITLE
Sixel image preview and previewSandbox.sh stretches pictures

### DISCUSSION THREAD

**Discussion by frere-jacques** [gokcehan/lf#1562] [Q&A]
I got the preview running for pictures in foot via sixel.
My script is nearly exactly the one on the page I only use different filter, because starting sixel calculation on raw files is not practical.

I wanted now to use the sandbox. It is working, but the pixtures are now stretched. I guess it is because the size parameter in the preview script is taken from variables $2 and $3 and I guess they are not correct within the sandbox script.

Any solution for that?

**Comment by joelim-work** [comment]
Possibly related: https://github.com/gokcehan/lf/pull/1211#issuecomment-1676512674

**Comment by CatsDeservePets** [comment]
Hey @frere-jacques!
There have been quite a few changes to `sixel` rendering inside `lf`. Does the bug still occur? Or has it been fixed?
